site/app/templates/admin/admin_gradeable/rubric/NumericRubric.twig

```diff
@@ -75,6 +75,9 @@
             else total_score += max_score;
         });
 
+        total_score = parseFloat(total_score.toFixed(2));
+        total_ec = parseFloat(total_ec.toFixed(2));
```

Comment:
No need to parse them back to a float since we only care about their string representation anyways. I'd recommend removing these two lines and modifying lines 81-82 to insert the toFixed there. Also would be good to change `.html` to `.text`.